specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/databases.json

```diff
@@ -0,0 +1,994 @@
+          },
+          "serviceTierAdvisors": {
+            "readOnly": true,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/ServiceTierAdvisor"
+            },
+            "description": "The list of service tier advisors for this database. Expanded property"
+          },
+          "transparentDataEncryption": {
+            "readOnly": true,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/TransparentDataEncryption"
+            },
+            "description": "The transparent data encryption info for this database."
+          },
+          "recommendedIndex": {
+            "readOnly": true,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/RecommendedIndex"
+            },
+            "description": "The recommended indices for this database."
+          },
+          "failoverGroupId": {
+            "readOnly": true,
+            "type": "string",
+            "description": "The resource identifier of the failover group containing this database."
+          },
+          "readScale": {
+            "type": "string",
+            "enum": [
+              "Enabled",
+              "Disabled"
+              ],
+            "description": "Conditional. If the database is a geo-secondary, readScale indicates whether read-only connections are allowed to this database or not. Not supported for DataWarehouse edition.",
+            "x-ms-enum": {
+              "name": "ReadScale"
+            }
+          },
+          "sampleName": {
+            "type": "string",
+            "enum": [
+              "AdventureWorksLT"
+              ],
+            "description": "Indicates the name of the sample schema to apply when creating this database. If createMode is not Default, this value is ignored. Not supported for DataWarehouse edition.",
+            "x-ms-enum": {
+              "modelAsString": true,
+              "name": "SampleName"
+            },
+            "x-ms-mutability": [ "create" ]
+          },
+          "zoneRedundant": {
+            "description": "Whether or not this database is zone redundant, which means the replicas of this database will be spread across multiple availability zones.",
+            "type": "boolean"
+          }
+        },
+        "description": "Represents the properties of a database."
+      },
+      "Database": {
+        "properties": {
+          "kind": {
+            "readOnly": true,
+            "type": "string",
+            "description": "Kind of database.  This is metadata used for the Azure portal experience."
+          },
+          "properties": {
+            "$ref": "#/definitions/DatabaseProperties",
+            "description": "The properties representing the resource.",
+            "x-ms-client-flatten": true
+          }
+        },
+        "allOf": [
+          {
+            "$ref": "#/definitions/TrackedResource"
```

Comment:
This ref does not resolve and is causing CI failures (same for elasticPools.json).